docs/src/main/sphinx/connector/join-pushdown-enabled-true.fragment

```diff
@@ -24,6 +24,11 @@ join pushdown:
     session property](session-properties-definition) is
     `join_pushdown_enabled`.
   - `true`
+* - `join-pushdown.with-expressions`
+  - Enable [join pushdown](join-pushdown) with complex expressions.
+    Equivalent [catalog session property](session-properties-definition) is
+    `complex_join_pushdown_enabled`.
+  - `true`
```

Comment:
@mosabua  follow-up

`aggregation-pushdown.enabled` & `topn-pushdown.enabled` (and their corresponding session properties) are nowhere to be found in the documentation (except in some old release notes). Please consider adding them in the existing documentation.

https://github.com/trinodb/trino/blob/dd02ee9c562e03aa3d761f1d69a1a679ed7bb972/plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java#L92-L107

https://github.com/trinodb/trino/blob/dd02ee9c562e03aa3d761f1d69a1a679ed7bb972/plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java#L92-L107

https://github.com/trinodb/trino/blob/dd02ee9c562e03aa3d761f1d69a1a679ed7bb972/plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java#L92-L107

Reply:
Uff .. yeah .. we have to add them to a fragment that we include in all jdbc connectors .. maybe @jhlodin and team can help here

Reply:
IMO those toggles should become hidden since there's now no reason to ever disable any of them. They are merely kill-switches now.

Reply:
So why are they implemented at all then? What use case are they needed for .. if they are a kill switch for debugging and other developers .. we should still document them.